### ISSUE TITLE
How about using Poltergeist?

### ISSUE THREAD

**Issue by JunichiIto** [everydayrails/rails-4-1-rspec-3-0#46]
You are using selenium-webdriver, but I don't use Selenium-webdriver day-to-day development. Recently I use Poltergeist.

Selenium-webdriver takes active window, so I don't like it so much. I think using headless browsers like Poltergeist is more practical approach. Do you have any plans to switch JS driver?

**Comment by JunichiIto** [comment]
I have a background for this request. I want to introduce Everyday Rails when talking about Poltergeist. If it uses Poltergeist, I could say "Please refer (or buy) Everyday Rails when you use it!"

**Comment by ruralocity** [comment]
I use Poltergeist as well. The main reason I used Selenium in the book is, when I first published the book, selenium-webdriver was included as a dependency of Capybara. If I switch it to Poltergeist, I introduce PhantomJS as an external dependency and potentially explain that installation process to readers. Since it varies from platform to platform I'd like to avoid that.

I also think it's useful for beginners to see what's happening when they use a JS driver in tests. Once that understanding is in place, it's just a matter of changing out a few lines in `rails_helper.rb` to use a headless driver instead.

I appreciate why you'd like to see the change, but at this time I think I'm going to keep it as-is. Let's revisit this when a major change to Rails or RSpec necessitates a new book version.
